README.md

```diff
@@ -13,7 +13,7 @@ pnpm dev # route types will be generated by next.js
 ## Known issues
 
 - Tool search does not set query parameters
-  - [(Shallow routing) updating search params causes server code to rerun.](https://github.com/vercel/next.js/issues/49668)
+- [(Shallow routing) updating search params causes server code to rerun.](https://github.com/vercel/next.js/issues/49668)
```

Comment:
This must be nested. I'm using React context instead of query parameters to avoid re-running server code 😀